ext/standard/tests/streams/locale.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Stream wrappers should not be locale dependent
```

Comment:
This test passes on Windows also without the patch for some reason.

Reply:
I think the locale files might depend on OS too, for some reason, I remember seeing something about that while looking at other tests